docker/x86_64.dockerfile

```diff
@@ -0,0 +1,37 @@
+ARG TAG=24.01-py3
+
+FROM nvcr.io/nvidia/tensorrt:${TAG} AS tensorrtx
+
+ENV DEBIAN_FRONTEND noninteractive
+
+# basic tools
+RUN apt update && apt-get install -y --fix-missing --no-install-recommends \
+sudo wget curl git ca-certificates ninja-build tzdata pkg-config \
+gdb libglib2.0-dev libmount-dev \
+&& rm -rf /var/lib/apt/lists/*
+RUN pip install --no-cache-dir yapf isort cmake-format pre-commit
+
+## override older cmake
+RUN find /usr/local/share -type d -name "cmake-*" -exec rm -rf {} + \
+&& curl -fsSL "https://github.com/Kitware/CMake/releases/download/v3.29.0/cmake-3.29.0-linux-x86_64.sh" \
+-o cmake.sh && bash cmake.sh --skip-license --exclude-subdir --prefix=/usr/local && rm cmake.sh
+
+## opencv contrib
+RUN git clone -b 4.x https://github.com/opencv/opencv_contrib.git
+
+## opencv
```

Comment:
Is it possible to install pre-built opencv?  Building from src will be slow.

Reply:
Add package installation as default method and leave the source build method commented for demo.